AxonDeepSeg/visualization/get_masks.py

```diff
@@ -17,17 +21,16 @@ def get_masks(path_prediction):
     myelin_prediction = prediction > 100
     myelin_prediction = myelin_prediction ^ axon_prediction
 
-    # get main path
-    path_folder, file_name = os.path.split(path_prediction)
-
-    tmp_path = path_prediction.split('_seg-axonmyelin')
-
-    # save the masks
-    if tmp_path[0].endswith('.png'):
-        tmp_path[0] = os.path.splitext(tmp_path[0])[0]
+    # We want to keep the filename path up to the '_seg-axonmyelin' part
+    path = path_prediction.parent
+    broken_filename = path_prediction.name.split('_seg-axonmyelin')[0]
+    # Extra check to ensure that the extension was removed
+    if broken_filename.endswith('.png'):
+        broken_filename = broken_filename.split('.png')[0]
 
-    imageio.imwrite(tmp_path[0] + '_seg-axon.png', axon_prediction.astype(int))
-    imageio.imwrite(tmp_path[0] + '_seg-myelin.png', myelin_prediction.astype(int))
+    # Save masks
+    imageio.imwrite(path.joinpath(broken_filename + '_seg-axon.png'), axon_prediction.astype(int))
```

Comment:
I think this joinpath was missed? Shouldn't we use the `path / filename` syntax?